components/Form/index.js

```diff
@@ -259,6 +304,30 @@ export default function Form({
       />
       <PackListContainer>
         <Label htmlFor="packingList">Packing List</Label>
+        <TemplateContainer>
+          <Select
+            id="template"
+            name="template"
+            onChange={(event) => setSelectedTemplate(event.target.value)}
+            value={selectedTemplate}
+            disabled={formDisabled}
+          >
+            <option value="" disabled>
+              Please select preset
+            </option>
+            <option value="Weekend">Weekend</option>
+            <option value="OneWeek">One week</option>
+            <option value="TwoWeeks">Two weeks</option>
+            <option value="ThreeWeeks">Three weeks</option>
```

Comment:
```suggestion
            <option value="weekend">Weekend</option>
            <option value="one week">One week</option>
            <option value="two weeks">Two weeks</option>
            <option value="three weeks">Three weeks</option>
```
The `value` attribute represents the value passed to the property with the respective `name` attribute (here: "template"), hence why it can be a regular string. This aids the readability of your database. :)